### PR TITLE
Add title and description to all pages

### LINKED ISSUE #368: Add title and description to pages
I added conditionals like:
``` 
{% if page and page.meta.title %}
      <meta name="twitter:title" content="{{ page.meta.title }}">
{% endif %}
```
because the template system was failing due to `page.meta.title` not existing on some pages, seems like it got stricter recently. Now there are generic defaults for these values. Longer term we should add these title/description values on all pages, similar to what is now at the top of the Roadmap page (and a few others).

_Originally posted by @zhyatt in https://github.com/nanocurrency/nano-docs/pull/366#issuecomment-674931770_

### PR DESCRIPTION
**Pull request by zhyatt**
Closes #368

### CODE REVIEW

File: docs/protocol-design/introduction.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Title: Protocol Design Introduction | Nano Documentation
+Title: Protocol Design Introduction
 Description: As the beginning of the living whitepaper, get details and background of the problems Nano aims to solve that other digital money hasn't been able to
```
Comment (guilhermelawless) [inline docs/protocol-design/introduction.md:2]: Title\Description should be lowercase

File: docs/what-is-nano/living-whitepaper.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-Title: Living Whitepaper | Nano Documentation
-Description: Overview of the living whitepaper on the Nano protocol and node design
+Title: Living Whitepaper
+Description: Overview of the living whitepaper on the Nano protocol and existing node implementation
```
Comment (guilhermelawless) [inline docs/what-is-nano/living-whitepaper.md:2]: Same here

File: docs/protocol-design/introduction.md (hunk 1 above)
Comment (zhyatt) [reply docs/protocol-design/introduction.md]: Done, good catch

File: docs/what-is-nano/living-whitepaper.md (hunk 2 above)
Comment (zhyatt) [reply docs/what-is-nano/living-whitepaper.md]: Done

**Review by guilhermelawless — APPROVED** [review]

**Pull request merged**